### DISCUSSION TITLE
FAQ: Where are the Pterodactyl server files stored?

### DISCUSSION THREAD

**Discussion by google-meet-meat** [pterodactyl/panel#2822] [General Community]
To find the location quickly, run this command for 0.7: `cat /srv/daemon/config/core.json | grep data` and this command for 1.0: `cat /etc/pterodactyl/config.yml | grep data`. You can also follow the manual instructions below.

In 0.7, the server files are stored in `/srv/daemon-data`. If you updated from 0.7 to 1.0, the server files are still stored in there unless manually moved.

However, if you have installed 1.0 without upgrading, the server files will be stored in `/var/lib/pterodactyl/volumes`.

**Comment by TekExplorer** [comment]
Note: if you upgraded from 0.7 to 1.0+, your files may still be in `/srv/daemon-data`!

**Reply by google-meet-meat** [reply]
> Note: if you upgraded from 0.7 to 1.0+, your files may still be in `/srv/daemon-data`!



> it depends on if you are on 0.7x or 1.0x. this could also of been changed when you set it up and upgrading. HOWEVER, normally in 1.0.x it is stored in `/var/lib/pterodactyl/volumes` and in 0.7x it is in `/srv/daemon-data/` if you upgraded from 0.7 to 1.0 it will be in `/srv/daemon-data/` unless you moved it

Thanks for the info, I have updated my submission.

**Comment by ElijahRus250** [comment]
it depends on if you are on 0.7x or 1.0x. this could also of been changed when you set it up and upgrading. HOWEVER, normally in 1.0.x it is stored in `/var/lib/pterodactyl/volumes` and in 0.7x it is in `/srv/daemon-data/` if you upgraded from 0.7 to 1.0 it will be in `/srv/daemon-data/` unless you moved it

**Comment by DaneEveritt** [comment]
You could also just run the command below rather than having to constantly guess at where they might be. It is also listed on the node configuration page in the Panel.

```
cat /etc/pterodactyl/config.yml | grep data
```

**Reply by google-meet-meat** [reply]
Thanks, I was not aware of that; I will incorporate that into my submission

**Comment by DevCallMeAJ** [comment]
Obviously, people who are experienced at Pterodactyl wouldn’t give out all the locations and they would say “go look at the docs.” Another location is ‘/var/www/pterodactyl’

**Reply by DaneEveritt** [reply]
That is for the Panel, not server files.
